googleapiclient/http.py

```diff
@@ -1257,6 +1257,8 @@ def _execute(self, http, order, requests):
 
     # Prepend with a content-type header so FeedParser can handle it.
     header = 'content-type: %s\r\n\r\n' % resp['content-type']
+    if not six.PY2:
+      content = content.decode('utf-8')
```

Comment:
May content have binary data?
If so, [surrogate escape](https://www.python.org/dev/peps/pep-0383/) may help us.
But I'm not sure about how FeedParser treat surrogate escaped text.